pkg/analyzer/analyzer_test.go

```diff
@@ -0,0 +1,59 @@
+package analyzer
+
+import (
+	"path/filepath"
+	"sort"
+	"testing"
+
+	"github.com/stretchr/testify/require"
+)
+
+func TestAnalyzer_Analyze(t *testing.T) {
+	tests := []struct {
+		name    string
+		paths   []string
+		want    []string
+		wantErr bool
+	}{
+		{
+			name:    "analyze_test_dir_single_path",
+			paths:   []string{filepath.FromSlash("../../test/fixtures/analyzer_test")},
+			want:    []string{"dockerfile", "cloudformation", "kubernetes", "openapi", "terraform", "ansible"},
```

Comment:
Can you try to add a scenario where we have the `Resources:` key word inside a terraform file?

Reply:
In a Terraform file, it will never go to the check content function, since terraform files are defined by the extension of the file, but I can add to other technologies to check